examples/3dtiles.html

```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>3d-tiles example</title>
+
+    <style type="text/css">
+            html {height: 100%}
+        body { margin: 0; overflow:hidden; height:100%}
+
+            div {
+                margin : auto auto;
+                width: 100%;
+                padding: 0;
+                height: 100%
+            }
+            #menuDiv {position: absolute; top:0px; margin-left: 0px;}
+
+
+        </style>
+        <meta charset="UTF-8">
+
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <script src="/node_modules/dat.gui/build/dat.gui.min.js"></script>
+    </head>
+    <body>
+        <div id="viewerDiv"></div>
+        <div id="viewerDiv2"></div>
+        <script src="/examples/GUI/GuiTools.js"></script>
+        <script src="/dist/itowns.js"></script>
+        <script src="/dist/debug.js"></script>
+        <script type="text/javascript">
+            /* global itowns,document,GuiTools*/
+
+            const positionOnGlobe = { longitude: -75.65044586, latitude: 40.062840764, altitude: 100000 };
+
+            // iTowns namespace defined here
+            const viewerDiv = document.getElementById('viewerDiv');
+
+            const menuGlobe = new GuiTools('menuDiv');
+            const globe = new itowns.GlobeView(viewerDiv, positionOnGlobe);
+            menuGlobe.view = globe;
+
+            itowns.Fetcher.json('layers/JSONLayers/Ortho.json').then(result => globe.addLayer(result));
+
+            // add 3d-tiles layer
+            const $3dTilesLayer = new itowns.GeometryLayer('3d-tiles');
+
+            $3dTilesLayer.preUpdate = (context, layer) => {
+                if(layer.root === undefined) {
+                    itowns.init3dTilesLayer(context, layer);
+                    return [];
+                }
+                itowns.pre3dTilesUpdate(context, layer);
+                return [layer.root];
+            };
+            $3dTilesLayer.update = itowns.process3dTilesNode(
+                itowns.$3dTilesCulling,
+                itowns.$3dTilesSubdivisionControl
+            );
+            $3dTilesLayer.url = 'http://localhost:8003/tilesets/TilesetWithDiscreteLOD/tileset.json';
+            $3dTilesLayer.protocol = '3d-tiles'
+            $3dTilesLayer.overrideMaterials = true;  // custom cesium shaders are not functional
+            $3dTilesLayer.type = 'geometry';
+            $3dTilesLayer.visible = true;
+
+            itowns.View.prototype.addLayer.call(globe, $3dTilesLayer);
```

Comment:
The GlobeView class [overrides the addLayer function](https://github.com/iTowns/itowns2/blob/master/src/Core/Prefab/GlobeView.js#L256). Instead of adding a layer to the scene, it attaches the layer to the globe tile layer.

If we want to add a layer that is independent from the tile layer, we have to use this syntax.